tests/test_launcher.py

```diff
@@ -450,3 +451,14 @@ def get_processor_count(solver):
     # https://github.com/ansys/pyfluent/issues/2624
     # with pyfluent.launch_fluent(additional_arguments="-t2") as solver:
     #     assert get_processor_count(solver) == 2
+
+
+def test_container_warning_for_host_mount_path(caplog):
+    if check_docker_support():
+        solver = pyfluent.launch_fluent()
+        assert (
+            "this path available as /mnt/pyfluent for the Fluent session running inside the container."
+            in caplog.text
+        )
+        assert pyfluent.EXAMPLES_PATH in caplog.text
+        assert os.getcwd() not in caplog.text
```

Comment:
I think the test should not assume that it will run in GitHub CI runners. I would say it would be best to make it more specific, configure the `container_dic` with the `host_mount_path` and `container_mount_path`, and assert both of those are in the warning message. 

For this test, can set `host_mount_path = os.getcwd()`. 

In this way, the test should work fine in both GitHub CI runners as well as when we run it locally in Windows or Linux machines outside of CI runners.

Reply:
Done. Thank you.